modules/xfem/include/base/XFEMFuncs.h

```diff
@@ -94,4 +94,38 @@ bool intersectSegmentWithCutLine(const Point & segment_point1,
  */
 Real crossProduct2D(const Point & point_a, const Point & point_b);
```

Comment:
Can we move this file into xfem/include/utils/XFEMFuncs.h, and maybe rename it to `XFEMHelperFunctions` or something like that?